en/docs/learn/analytics/analyzing-apim-statistics-with-batch-analytics/using-geo-location-based-statistics/configuring-geolocation-based-statistics.md

```diff
@@ -1,26 +1,20 @@
-# Configuring Geolocation Based Statistics
+# Configuring Geo Location Based Statistics
 
 !!! note
     This documentation uses MySQL as an example for configuring the GEO_LOCATION_DATA database.
 
 !!! info
     In order to generate Geolocation based statistics, you need to pass an `x-forwarded-for` header with the relevant IP in the API request.
 
-1.  Download the Geolocation data from [here](https://svn.wso2.org/repos/wso2/people/tharindua/geodata/Geolocation%20Data.tar.gz).
-2.  Unzip the file you downloaded.
-
-    !!! info
-        If you have Geo Location Dataset already dowloaded, you can update that dataset by following [Updating Geo Location Data Set](../updating-geo-location-data-set/).
-
-
-3.  Create the `GEO_LOCATION_DATA` database by executing one of the scripts in the `Geolocation Data/dbscripts` directory. In this example, `mysql.sql` is executed.
+1.  Use the Geo Location dataset that you created [here](../creating-geo-location-data-set/).
```

Comment:
Check this link. It should start with {{base_path}} or if u are linking to the same page it will start with #

Reply:
Fixed in d9c29843ef4201cb9eca588053bea4aaecd174e0